packages/cli/templates/react/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "module": "ESNext",
-    "moduleResolution": "Node",
+    "moduleResolution": "bundler",
```

Comment:
This was the biggest headache for me. After implementing this,

```js
import { useFieldPlugin } from '@storyblok/field-plugin/react'
```

it seemed to work fine, but TypeScript actually couldn't resolve its type definition, while the build script already correctly generated the type definition. 

Then what went wrong? It's because of TypeScript's limitation. It was unable to resolve the type definition in this situation. Now in TypeScript v5, it's possible if you put `bundler` as `moduleResolution` like this.

You can read more in detail: https://dev.to/ayc0/typescript-50-new-mode-bundler-esm-1jic

Reply:
thank you for the explanation, this one looks like it cause quite some headache :D

Reply:
@BibiSebi yes it was! Huge shoutout to @alvarosabu who helped me fix this issue!